services/auth/office/README.md

```diff
@@ -0,0 +1,43 @@
+# Helsingborg IO SLS - Office Authorizer Service
+
+
+## Table of Contents
+- [Table of Contents](#table-of-contents)
+- [About Token](#about-office-authorizer)
+- [Getting Started](#getting-started)
+  - [Do first](#do-first)
+  - [Run local](#run-local)
+  - [Deploy](#deploy-and-run-on-aws)
+
+
+## About Office Authorizer
+
+The Office Authorizer service purpose is to provide an authorization method for Microsoft Office users that is authenticated with
+Azure AD.
+
+
+## Getting Started
+
+
+### Do first
+Read the global requierments for this repo, can be found [here](https://github.com/helsingborg-stad/helsingborg-io-sls-api/blob/dev/README.md)
```

Comment:
The README copy-paste typo strikes again 😄 
requierments -> requirements

Reply:
:/ , will fix!